pkg/credentialprovider/plugin/plugin.go

```diff
@@ -372,7 +389,7 @@ func (e *execPlugin) ExecPlugin(ctx context.Context, image string) (*credentialp
 		return nil, errors.New("apiVersion from credential plugin response did not match")
```

Comment:
this is pre-existing, but it would be helpful to include the expected and actual version in the error